### PR TITLE
Change formatting of `mc stat`

### PR DESCRIPTION
**Pull request by taran-p**
## Description
Make changes requested in #4352 

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

File: cmd/stat.go [hunk 1]
```diff
@@ -489,23 +506,25 @@ func prettyPrintBucketMetadata(info BucketInfo) string {
 	fmt.Fprintf(&b, "%2s%s", placeHolder, "Location: ")
 	fmt.Fprint(&b, console.Colorize("Generic", info.Location))
 	fmt.Fprintln(&b)
-	fmt.Fprintf(&b, "%2s%s", placeHolder, "Policy: ")
+	fmt.Fprintf(&b, "%2s%s", placeHolder, "Annonymous: ")
```
Comment (klauspost) [inline cmd/stat.go:509]:
```suggestion
	fmt.Fprintf(&b, "%2s%s", placeHolder, "Anonymous: ")
```

File: cmd/stat.go [hunk 2]
```diff
@@ -453,10 +465,15 @@ func prettyPrintBucketMetadata(info BucketInfo) string {
...
-		fmt.Fprint(&b, console.Colorize("Key", "\n\tKey ID: "))
-		fmt.Fprint(&b, console.Colorize("Value", info.Encryption.KeyID))
+		if info.Encryption.Algorithm == "aws:kms" {
+			fmt.Fprint(&b, console.Colorize("Key", "\n\tKey Type: "))
+			fmt.Fprint(&b, console.Colorize("Value", "SSE-KMS"))
+			fmt.Fprint(&b, console.Colorize("Key", "\n\tKey ID: "))
+			fmt.Fprint(&b, console.Colorize("Value", info.Encryption.KeyID))
+		} else {
+			fmt.Fprint(&b, console.Colorize("Key", "\n\tKey Type: "))
+			fmt.Fprint(&b, console.Colorize("Value", "SSE-S3"))
```
Comment (klauspost) [inline cmd/stat.go:475]: Isn't there also `SSE-C`? Maybe this instead?

```suggestion
			fmt.Fprint(&b, console.Colorize("Value", strings.ToUpper(info.Encryption.Algorithm)))
```

Comment (taran-p) [reply cmd/stat.go:475]: I believe SSE-C can't be set on a bucket, so only KMS and S3 are options. S3 algorithm is AES256, so have it print as SSE-S3 when printing as a string for simplicity.

**Review by poornas — APPROVED** [review]

File: cmd/stat.go (hunk 2 above)
Comment (harshavardhana) [reply cmd/stat.go:475]: Keep it generic use the value from `info.Encryption.Algorithm`

**Review by klauspost — APPROVED** [review]

**Pull request merged**